core/sequencer/server.go

```diff
@@ -323,21 +323,22 @@ func (s *Server) ApplyRevisions(ctx context.Context, in *spb.ApplyRevisionsReque
 		return nil, err
 	}
 
-	for i := int64(1); i < int64(s.LogPublishBatchSize); i++ {
+	i := int64(1)
+	for ; i <= int64(s.LogPublishBatchSize); i++ {
```

Comment:
How about looping from 0, while `revs < int64(...)`, and setting `Revision: highestApplied+revs+1` below?

Reply:
Sure